recipes/folly/all/conanfile.py

```diff
@@ -64,9 +64,10 @@ def configure(self):
         if self.version == "2020.08.10.00" and self.settings.compiler == "clang" and self.options.shared:
             raise ConanInvalidConfiguration("Folly could not be built by clang as a shared library")
 
-    # Freeze max. CMake version at 3.16.2 to fix the Linux build
     def build_requirements(self):
-        self.build_requires("cmake/3.16.2")
+        # Freeze max. CMake version at 3.16.2 to fix the Linux build
+        if Version(self.version) <= "2020.08.10.00":
+            self.build_requires("cmake/3.16.2")
 
     def requirements(self):
```

Comment:
Please bump openssl

Reply:
It's the only one I know off hand... and it will not require extra work 👍 

There could be more but it might break older versions 🤷